### DISCUSSION TITLE
Tune Detections page not displaying all suppressions

### DISCUSSION THREAD

**Discussion by BradMic** [Security-Onion-Solutions/securityonion#13949] [2.4] [answered]
### Version

2.4.110

### Installation Method

Security Onion ISO image

### Description

other (please provide detail below)

### Installation Type

Distributed

### Location

on-prem with Internet access

### Hardware Specs

Meets minimum requirements

### CPU

28

### RAM

90GB

### Storage for /

130GB

### Storage for /nsm

12TB

### Network Traffic Collection

tap

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

On the SOC Alerts page, I left-click on an alert and select Tune Detection for rule 2013504 and then click the Tuning tab, it only shows 10 supressions but if I look at the file, /opt/so/saltstack/local/salt/suricata/thresholding/sids.yaml, I can all the supressions I added. Is this page limited to only display 10?

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by defensivedepth** [comment, marked as answer]
I have confirmed this is a bug and created an issue to track it - https://github.com/Security-Onion-Solutions/securityonion/issues/13950

Should be fixed in the next release.

**Reply by BradMic** [reply]
Thank you!
